easybuild/easyblocks/c/code_saturne.py

```diff
@@ -0,0 +1,167 @@
+##
+# Copyright 2009-2018 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild. If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing Code_Saturne,
+implemented as an easyblock.
+
+@author: Metin Cakircali (Juelich Supercomputing Centre, FZJ GmbH)
+"""
+
+import os
+
+from easybuild.framework.easyblock import EasyBlock
+from easybuild.framework.easyconfig.default import CUSTOM
+from easybuild.tools.filetools import apply_regex_substitutions
+from easybuild.tools.run import run_cmd
+from easybuild.tools.systemtools import get_shared_lib_ext
+
+import easybuild.tools.environment as env
+
+
+class EB_Code_underscore_Saturne(EasyBlock):
```

Comment:
@surak Wouldn't it be better to derive from `ConfigureMake`, that would simplify things below quite a bit (there's no need to customize `build_step` and `install_step` for example in that case...